supervision/annotators/core.py

```diff
@@ -1011,37 +1011,42 @@ class LabelAnnotator(BaseAnnotator):
     def __init__(
         self,
         color: Union[Color, ColorPalette] = ColorPalette.DEFAULT,
-        text_color: Color = Color.WHITE,
+        text_color: Union[Color, ColorPalette] = ColorPalette.LEGACY,
```

Comment:
Small suggestion:  I think we should keep white as default because it changes readability and also change default behavior, up until this version we put "white" and when people see different color It can create some confusion. It would be better keep "Color.WHITE" as default but keep type change is   "Union[Color, ColorPalette]" in case of multi color change.

Reply:
@onuralpszr, yess certainly. I changed this because it was overlapping with ColorPalette.DEFAULT and the labels weren't visible but I will change it to White now.

Also, couldn't understand this `but keep type change is "Union[Color, ColorPalette]" in case of multi color change.` Can you explain it further? Thanks

Reply:
I might be wrong on this but I think there is no such method. Although, there is `COLOR.WHITE`

Reply:
@Bhavay-2001  ah right I miss typed yes Color.WHITE